elections/steering/2023/election.yaml

```diff
@@ -0,0 +1,17 @@
+name: 2023 Steering Committee Election
+organization: Kubernetes
+start_datetime: 2023-08-29 00:00:01
```

Comment:
I _think_ that this field means that I can rename the file from election-template.yaml to election.yaml now, even though the voters.yaml isn't ready yet, because in theory the election doesn't go "live" until this start_datetime.

Reply:
> BTW, the election will show in elekto as soon as you merge. Just nobody will be able to vote, and there will be no candidates.

Ohhhh, okay. So possibly I don't want to move the template file to its "live file" name yet? I don't want to create confusion.

Reply:
@jberkus I have tried changing that file back to its template name, so as to avoid Early Empty Election Launch. Perhaps that resolves the immediate issue?

Reply:
(I do have a voters.yaml but if the specific empty list format Elekto would need is not what the example file comes with, perhaps we should adjust the example?)

Reply:
No no no, you misunderstand me.  Having the blank election up is a *good* thing.